feature-libs/user/account/components/login-form/login-form.component.html

```diff
@@ -70,12 +70,3 @@
     {{ 'loginForm.signIn' | cxTranslate }}
   </button>
 </form>
-
-<ng-template #requiredAsterisk>
-  <abbr
-    *cxFeature="'a11yRequiredAsterisks'"
-    class="text-decoration-none"
-    title="{{ 'common.required' | cxTranslate }}"
-    >*</abbr
-  >
-</ng-template>
```

Comment:
Good on you from cleaning it up, somebody must have left it here while removing the asterisks from the login page. 🤷